patch-2.2.1.patch

```diff
@@ -0,0 +1,11 @@
+--- tool/rbinstall.rb   2015-03-07 21:59:28.834830008 +1000
```

Comment:
Is this intentional?

Reply:
corrected, I was patching ruby to get it to install